src/static/helpers/environment.ts

```diff
@@ -1,3 +1,11 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {FetchFunction} from 'lib/clientConfig';
```

Comment:
```suggestion
import type {FetchFunction} from '../clientConfig';
```
1.`import type` tells typescript that there's no JS with side effects (e.g. modifying globals) in the imported files, so typescript can remove the import when compiling.
2. Imports from `lib` break for me locally 😐

Reply:
Oh, good point! If this works better, I can merge this in.